common/pxc-db/values.yaml

```diff
@@ -0,0 +1,389 @@
+# Default values for percona-xtradb-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+linkerd:
+  enabled: true
+
+owner_info:
+  enabled: false
```

Comment:
I don't think we need it, because owner_info comes as dependency from openstack charts.

Reply:
Removed this dependency, because it's no longer needed in labs